classes/mail/Mailer.php

```diff
@@ -117,6 +117,12 @@ public function send($view, array $data = [], $callback = null)
             /** @var Mailable $view */
             $view->setData();
         }
+
+        // Application is set to sandbox mode and will not sent any emails
+        if (Config::getVar('general', 'sandbox', false)) {
+            return null;
```

Comment:
I would suggest an `error_log` call wherever an action is skipped due to sandbox mode. (And mostly stylistic, but maybe use `return;` rather than `return null;` while you're at it -- this function doesn't return anything.)

Reply:
done .